pkg/nimble/scanner/include/nimble_scanner.h

```diff
@@ -68,6 +68,9 @@ int nimble_scanner_init(const struct ble_gap_disc_params *params,
 
 /**
  * @brief   Start scanning using timing parameters configured on initialization
+ *
+ * @note    Scanning will run for ever unless stopped or unless a different
+ *          scan duration is set with @ref nimble_scan_set_scan_duration
```

Comment:
`nimble_scan_set_scan_duration` does not exist in doxygen

Reply:
addressed